homework-submission/week3/client/src/App.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './App.css';
+import Spinner from './components/spinner/index';
+import HomeworkWeek3 from './components/HomeworkWeek3/index';
+
+class App extends React.Component {
+  state = {
+    data: [],
+    isLoading: true,
+  };
+
+  componentDidMount() {
+    fetch('https://uinames.com/api/?amount=10')
```

Comment:
What if the request fails? Make sure you add a `catch` for the errors.

Reply:
Yeah, I forgot to handle the errors :pray: done.